### PR TITLE
Update login.inc.php

### PR DESCRIPTION
**Pull request by aspiringstar**
Passed non-invalid (eg-if wrongpassword is encountered, then username and roll no is passed) entries through get method inorder to make login page interface more user-friendly

### CODE REVIEW

File: includes/login.inc.php [hunk 1]
```diff
@@ -104,7 +104,7 @@
                 if($row = mysqli_fetch_assoc($result)){
                     $pwdCheck = password_verify($password,$row['pwd']);
                     if($pwdCheck == false){
-                        header("Location: ../login.php?login-type=staff&error=wrongpassword");
+                        header("Location: ../login.php?login-type=staff&error=wrongpassword&username=$username&roll=$rollno");
```
Comment (ayan-biswas0412) [inline includes/login.inc.php]: From where you get roll no for staffs ?

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

**Comment by ayan-biswas0412** [comment]
Resolve this and just add commit push no need to do a second pull request

File: includes/login.inc.php (hunk 1 above)
Comment (aspiringstar) [reply includes/login.inc.php]: oh my bad, there will not be any changes for staff.

Comment (aspiringstar) [reply includes/login.inc.php]: only students would be changed.

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
If the user inputs his correct username + wrong roll no + right password ,
then user gets logged in which is a potential risk so i am closing this issue .

**Comment by ayan-biswas0412** [comment]
There is no need in this work because it leds to an potential security risk.
You may work in the issue #52

**Pull request closed without merging**